homeassistant/components/dsmr/sensor.py

```diff
@@ -32,8 +32,8 @@
 ICON_POWER_FAILURE = "mdi:flash-off"
 ICON_SWELL_SAG = "mdi:pulse"
 
-# Smart meter sends telegram every 10 seconds
-MIN_TIME_BETWEEN_UPDATES = timedelta(seconds=10)
+# Smart meter sends telegram every 10 seconds or every second for DSMR 5
+MIN_TIME_BETWEEN_UPDATES = timedelta(seconds=1)
```

Comment:
Where is this constant used? Where is it changed to 10 seconds for other DSMR versions?

Reply:
Hmm not used indeed. Removing it!

Reply:
I think this file is actually missing the following: `@Throttle(MIN_TIME_BETWEEN_UPDATES)` before the update method.

Reply:
Here is an example:
https://github.com/home-assistant/home-assistant/blob/b41480ae466a99c977e138d335e552107e9865ac/homeassistant/components/metoffice/sensor.py#L192-L193

Reply:
@dupondje could you implement it like this? And please add the different intervals to cover all versions.

Reply:
I could try :)
Adding the @Throttle before def update_entities_telegram(telegram): ?

Also I don't see how to update the MIN_TIME_BETWEEN_UPDATES depending on the DSMR_VERSION

Reply:
> Adding the @Throttle before def update_entities_telegram(telegram): ?

Yes exactly :)

> Also I don't see how to update the MIN_TIME_BETWEEN_UPDATES depending on the DSMR_VERSION

That is okay, just keep it at 10 seconds then.

Reply:
Well keeping it at 10 seconds loses data on DSMR 5.0 :)
Anyway, did a change, should be fine now!